kvdb-memorydb/Cargo.toml

```diff
@@ -1,16 +1,20 @@
 [package]
 name = "kvdb-memorydb"
-version = "0.7.0"
+version = "0.8.0"
 authors = ["Parity Technologies <admin@parity.io>"]
 repository = "https://github.com/paritytech/parity-common"
 description = "A key-value in-memory database that implements the  `KeyValueDB` trait"
 license = "MIT OR Apache-2.0"
 edition = "2018"
 
 [dependencies]
-parity-util-mem = { path = "../parity-util-mem", version = "0.7", default-features = false, features = ["std"] }
-parking_lot = "0.10.0"
-kvdb = { version = "0.7", path = "../kvdb" }
+parity-util-mem = { path = "../parity-util-mem", version = "0.8", default-features = false, features = ["std"] }
+parking_lot = "0.11.1"
+kvdb = { version = "0.8", path = "../kvdb" }
 
 [dev-dependencies]
-kvdb-shared-tests = { path = "../kvdb-shared-tests", version = "0.5" }
+kvdb-shared-tests = { path = "../kvdb-shared-tests", version = "0.6" }
+
+[features]
+default = []
+wasm-bindgen = ["parking_lot/wasm-bindgen"]
```

Comment:
This is required because of https://github.com/Amanieu/parking_lot/blob/37ceeb36a1b0ef9cc0176ef0c9e35b8562002656/Cargo.toml#L31

Reply:
FYI, I didn't add this to:

-  `parity-util-memory` because it's an optional dependency
- `kvdb-rocksdb` because it doesn't compile for WASM

Reply:
This is not ideal, because we'd want to eventually get rid of `kvdb-memory` from `kvdb-web`, but that would require making `kvdb` async. 
What will likely happen is that `kvdb-web` will be deprecated (I think it's no longer used already?) in favor of what substrate-lite uses (based on localStorage?). And this feature of `kvdb-memorydb` will become useless.

However, I think `kvdb-memorydb` is mostly used for tests, so it's fine I guess ¯\_(ツ)_/¯

Reply:
ok, we could do another super ugly hack, just to bring in `parking_lot` as a dependency in `kvd-web` with the `wasm-bindgen` feature then we don't have to have it in `kvdb-memory` because of how features work.

Reply:
yeah, that's what I did when I wrote
> although in a hacky way

I think it's fine as is though. `kvdb-memorydb` could be in theory used in as `wasm-bindgen` environment

Reply:
> just to bring in `parking_lot` as a dependency in `kvd-web` with the `wasm-bindgen` feature then we don't have to have it in `kvdb-memory` because of how features work.

This is terrible but let's do it anyway. We need a fat-ass comment in `kvdb-web/Cargo.toml` reminding us why it's here and to get rid of it as soon as possible.